server/src/model/page.ts

```diff
@@ -170,10 +170,12 @@ export class PageNode extends Fileish {
       }
     })))
 
+    const introClasses = ['introduction', 'unit-opener']
```

Comment:
I think a unit-opener can only occur inside a unit and a chapter introduction can only exist in a chapter. Lemme ask why the term "unit-opener" was used instead of "introduction" inside a Unit